### PR TITLE
Replace SideTable with WeakMap

### PR DESCRIPTION
**Pull request by arv**

### CODE REVIEW

**Comment by arv** [comment]
This time I believe I managed to update the conf files as well.

File: build.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [
     "../observe-js/src/observe.js",
-    "src/sidetable.js",
+    "../WeakMap/waekmap.js",
```
Comment (dfreedm) [inline build.json]: "weakmap"

**Comment by dfreedm** [comment]
LGTM after fixing build.json

**Comment by arv** [comment]
That is why code reviews are awesome.

**Pull request merged**